pricegraph/src/api/price_estimation.rs

```diff
@@ -33,7 +33,7 @@ impl Pricegraph {
             orderbook.fill_optimal_transitive_order_if(inverse_pair, |flow| {
                 last_exchange_rate = Some(flow.exchange_rate);
                 false
-            })?;
+            });
```

Comment:
Interesting. Why does the linter not yell at you to consume this Option?

Reply:
`Result`s are `#[must_use]` but not Options. If this function was pure it might be a warning (not sure) but in this it but in this case it has side effects so it is reasonable to ignore the return value.

Reply:
I made the code a little more self-documenting.